Revamp makefile (#1402)
jellyfin/jellyfin-roku: docs/DEVGUIDE.md

cd jellyfin-roku
```

## Install Dependencies

You'll need [`npm`](https://nodejs.org), the default package manager for Node.js.

> For Debian or Ubuntu, install it via:

Reviewer: ```suggestion
To install on Debian based distros:
```

Maybe this is nitpicking but if feels weird to call out these two distros and not Mint or any other debian based distros.

Another solution would be to change the text to read `To install on Linux:` and then have a line for each major package manager but this might be overkill and I'm not even sure if there's an npm package available for each of them.
```bash
sudo apt-get install npm # Debian based distros

```

```bash
sudo apt-get install npm
```

> Then, use `npm` to install more dependencies

Reviewer: ```suggestion
Then, use `npm` to install dependencies
```

More doesn't sound right to me. `app dependencies` or `required dependencies` would also work

Note: You only have to run this command once if you are not a developer. The Jellyfin channel will still be installed after rebooting your Roku device.

### Bug/Crash Reports

Did the app crash? Find a nasty bug? Use this command to view the error log and [report it to the developers](https://github.com/jellyfin/jellyfin-roku/issues):

```bash
telnet ${ROKU_DEV_TARGET} 8085
```

To exit telnet: `CTRL + ]` and then type `quit + ENTER`

You can also take a screenshot of the app to augment the bug report.

```bash
make screenshot
```

## Committing

Before committing your code, please run:

```bash
npm run lint
```

And fix any encountered issue.

## (Optional) Update Images

> This repo already contains all necessary images for the app. This script only needs to be run when the [official Jellyfin images](https://github.com/jellyfin/jellyfin-ux) are changed to allow us to update the repo images. It only works using the [Command Line Method](method-2-command-line).

Reviewer: ```suggestion
> NOTE: This script is only compatible with [Method 2: Command Line](method-2-command-line).
This repo already contains all necessary images for the app. This script only needs to be run when the [official Jellyfin images](https://github.com/jellyfin/jellyfin-ux) are changed to allow us to update the repo images.
```

If you leave it as a top level, I think this is more clear.

Author: moved to Method 2 block and removed the line referencing the method.
